Student/Student/Courses/Syllabus/SyllabusViewController.swift

```diff
@@ -0,0 +1,187 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class SyllabusViewController: UIViewController {
+
+    @IBOutlet weak var menuBorder: UIView!
+    @IBOutlet weak var menuBorderHeight: NSLayoutConstraint!
+    @IBOutlet weak var menu: HorizontalMenuView!
+    @IBOutlet weak var scrollView: UIScrollView!
+    var presenter: SyllabusPresenter!
+    var titleView: TitleSubtitleView!
+    var courseID: String = ""
+    var color: UIColor?
+    var syllabus: CoreWebView?
+    var assignments: AssignmentListViewController?
+
+    enum MenuItem: Int {
+        case syllabus, assignments
+    }
+
+    static func create(courseID: String) -> SyllabusViewController {
+        let vc = loadFromStoryboard()
+        vc.courseID = courseID
+        return vc
```

Comment:
I'd feel better if the `presenter` was set in here instead of in `viewDidLoad`. Both because that's how we've been doing it everywhere else and because it's a force unwrapped `SyllabusPresenter!`. I'm worried about someone trying to access the presenter before `viewDidLoad` gets called.